source/fundamentals/crud/read-operations/cursor.txt

```diff
@@ -5,4 +5,140 @@ Access Data From a Cursor
 .. default-domain:: mongodb
 
 Read operations that return multiple documents do not immediately return
-all values matching the query.
+all values matching the query. Because a query can potentially match
+large sets of documents, these operations rely upon a
+:java-docs:`FindIterable
+<apidocs/mongodb-driver-sync/com/mongodb/client/FindIterable.html>`.
```

Comment:
It's going to be confusing if this page seems to be limited to using FindIterable.  I thought the idea was to make it a general page about iterating cursors, and use `find` as the example.  But everything here applies equally to the other methods that return instances of `MongoIterable`, e.g. aggregate, listCollections.

I think this needs to be generalized.

Reply:
I originally wrote only about methods within MongoCursor since the page is called "Access Data From a Cursor".  But with the terminal methods added in, it shifted its focus to a FindIterable.  

I've generalized the introduction a bit more. Let me know what you think of the current edits.

Reply:
I like the edits.  Please consider adding a note indicating that everything below applies equally to other iterables, esp AggregateIterable